### DISCUSSION TITLE
CLI alias

### DISCUSSION THREAD

**Discussion by ivodolenc** [prettier/prettier#15172] [Ideas]
Hi, I notice that when we use CLI commands, sometimes it can be useful when we can use aliases, i.e. abbreviations.

For example, the command `--config` may have an alias `-c` which is standard among other tools such as eslint, rollup, etc.

Also, if we have a longer path, it can improve readability.

### Current:

```json
{
  "scripts": {
    "format": "prettier --config .config/prettier.config.js --write ."
  }
}
```

### Suggestion:

```json
{
  "scripts": {
    "format": "prettier -c .config/prettier.config.js --write ."
  }
}
```

**Comment by kachkaev** [comment]
👋 @ivodolenc! `prettier -c` is an alias for `prettier --check` so this shortcut cannot be used for config. See `prettier --help` for details.

**Comment by ivodolenc** [comment]
Ah, I hadn't noticed, thanks for pointing that out @kachkaev 

Using other popular tools, you'd think `-c` is alias for `--config`.
